Neos.Flow/Classes/Mvc/Routing/IdentityRoutePart.php

```diff
@@ -55,8 +55,7 @@ class IdentityRoutePart extends DynamicRoutePart
     /**
      * The object type (class name) of the entity this route part belongs to
      *
-     * @var string
-     * @psalm-var class-string
+     * @var class-string
```

Comment:
What does this do to reflection? Might not be important here, but e.g. in a entity this would probably be breaking.

Reply:
yes in entities this will not work at all.

My rule is to use those modern annotations everywhere except in entities and controllers as flow will crash otherwise.

And in this case i like to leave it here.